### ISSUE TITLE
Clear Cast doesn't amp R

### ISSUE THREAD

**Issue by Xhulhorac** [ChalkyBrush/roshpit-bug-tracker#15]
Clear Cast's cooldown refresh and instant-cast aspect works, but the said 300% damage increase does not.

**Comment by redledcat** [comment]
Works for me.

![20171010151603_1](https://user-images.githubusercontent.com/32621322/31386213-499e3e68-adc6-11e7-956a-d3cd5019116c.jpg)
165k  just Rlv 7
231k  lv1 E +R lv 7 (x1.4)
660k lv7 E + R lv 7 (x4)
![20171010152059_1](https://user-images.githubusercontent.com/32621322/31386216-4d598742-adc6-11e7-9bd8-a67f72acdcff.jpg)

**Comment by ChalkyBrush** [comment]
xhul could it be the problem you are describing is with her arcana? (Ice Tornado)

**Comment by Xhulhorac** [comment]
https://www.roshpit.ca/players/118352521/heroes/689385

The result of Clear Cast not amping R was acquired with this hero, these items.

Neither R or R2 had any difference in damage.

**Comment by redledcat** [comment]
Well, I've tested base ability damage % buff interactions and T7 glyph, both works for R ability itself. 
In spite of this, R2 remains constant number not based on changes in R damage, i.e. I increased and desreased my base ability damage, but R2 got no pros out of it, but logically if R damage grows, then R2 damage should grow linearly. And R2 always was affected by Clearcast after first Clearcast in the game got proc. I've got lesser values without E casted in game, and constant bigger value after first clearcast.
Also: damage is strange. R gets 4539 on R lv1, E lv7, and T7 glyph. And, yeah, it deals 999 damage instead of listed 1000 on lv1 without any buffs on dummy.
